added Crowboard

doesntfazer opened the pull request:
This is my Crowboard Keyboard. I would like to add it as a supported keyboard.

kdb424 commented:
I saw this board on Reddit, and love it! I'd love to upstream it, but we request that you offer at least one place to get this keyboard, be it your store, or if it's open source, a link to where to get the files, like a git repo. Please check out other keyboard files like the crkbd, but otherwise, am very happy to get this merged once the tests and that are resolved. Most of the linting that the broken check is failing on should be fixed by running `make fix-formatting fix-isort test` and if that doesn't report issues, it's all fixed on that side.

doesntfazer commented:
> I saw this board on Reddit, and love it! I'd love to upstream it, but we request that you offer at least one place to get this keyboard, be it your store, or if it's open source, a link to where to get the files, like a git repo. Please check out other keyboard files like the crkbd, but otherwise, am very happy to get this merged once the tests and that are resolved. Most of the linting that the broken check is failing on should be fixed by running `make fix-formatting fix-isort test` and if that doesn't report issues, it's all fixed on that side.

I have updated it with the store, and I added the github repo.

kdb424 commented:
Just need to get tests passing, and handle my line comments, and should be able to merge it in. If you need help with anything, let us know.

doesntfazer commented:
I'm also a little confused on the make part. Where would I run that command?

>  `make fix-formatting fix-isort test` and if that doesn't report issues, it's all fixed on that side.

kdb424 commented on boards/crowboard/main.py:
```diff
@@ -0,0 +1,103 @@
+from kmk.kmk_keyboard import KMKKeyboard
+
+=======
+from kb import KMKKeyboard
```
This is the one that should be kept

kdb424 commented on boards/crowboard/main.py:
```diff
@@ -0,0 +1,103 @@
+<<<<<<< Updated upstream
+=======
+
+# Uncomment for Trackball
```
This can be stored in comments, but I removed it just to clean up what I thought were possible errors. Seems like this can be kept as comments.

kdb424 reviewed (approved):
Looks great. Thanks for sticking with me for that!

kdb424 commented on boards/crowboard/main.py:
```diff
@@ -0,0 +1,93 @@
+from kb import KMKKeyboard
+
+from kmk.keys import KC
+from kmk.kmk_keyboard import KMKKeyboard
```
This import is wrong. This overrides the import from above. Sorry that I missed this.
